### PR TITLE
added front-end for invitation link

### PR DESCRIPTION
**Pull request by alantylam**
wasn't sure if i should make a new vue for the invitation link.

### CODE REVIEW

**Comment by alantylam** [comment]
inviteLink_before
inviteLink_after png

**Comment by alantylam** [comment]
I need to make to into its own component.

**Comment by sam-ao** [comment]
I just changed the formatter we're using to prettier so please run the following commands to format your code:
npm install
npx pretty-quick

File: frontend/src/modules/lobby/components/InvitationLink.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+</template>
+
+<script>
+
+export default {
+  name: "InvitationLink",
+  data: function() {
+    return {
+        // inviteLink: window.location.href
+        inviteLink: "http://" + document.location.hostname + ":" + document.location.port + "/#/" + this.stage + "/" + this.lobbyID
```
Comment (gordon-cheung) [inline frontend/src/modules/lobby/components/InvitationLink.vue:20]: Have you looked into using this.$router to get he inviteLink?

File: frontend/src/modules/lobby/components/InvitationLink.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    };
+  },
+  props: {
+    lobbyID: String,
+    stage: String
+  },
+  methods: {
+    copyInviteLink: function() {
+      let linkToCopy = document.querySelector('#invitation-link');
```
Comment (gordon-cheung) [inline frontend/src/modules/lobby/components/InvitationLink.vue:29]: What does this do?

File: frontend/src/modules/lobby/view/Lobby.vue [hunk 3]
```diff
@@ -8,13 +8,15 @@
       <b-button class="exit-button" variant="danger" v-on:click="exitToHome">Exit Lobby</b-button>
     </div>
     <PlayerList :players="players" />
+    <InvitationLink :lobbyID="lobbyId" :stage="stage" />
```
Comment (gordon-cheung) [inline frontend/src/modules/lobby/view/Lobby.vue:11]: I like how you considered reusseability in this scenario

File: frontend/src/modules/lobby/components/InvitationLink.vue (hunk 2 above)
Comment (alantylam) [reply frontend/src/modules/lobby/components/InvitationLink.vue:29]: it copies the link from the <input>

File: frontend/src/modules/lobby/components/InvitationLink.vue (hunk 1 above)
Comment (alantylam) [reply frontend/src/modules/lobby/components/InvitationLink.vue:20]: I did some google when i was doing this part and i can't really get this to work. I will try to do more digging. but yea i think what i have here is super janky.

File: frontend/src/modules/lobby/view/Lobby.vue [hunk 4]
```diff
@@ -3,18 +3,24 @@
...
     <div class="lobby-buttons">
-      <b-button class="lobby-button" variant="success" v-on:click="goToScribble">Start game</b-button>
+      <b-button class="lobby-button" variant="success" v-on:click="goToScribble"
+        >Start game</b-button
+      >
       <Nickname @onChangeNickname="onChangeNickname" />
-      <b-button class="exit-button" variant="danger" v-on:click="exitToHome">Exit Lobby</b-button>
+      <b-button class="exit-button" variant="danger" v-on:click="exitToHome"
+        >Exit Lobby</b-button
+      >
```
Comment (enochtsang) [inline frontend/src/modules/lobby/view/Lobby.vue:12]: This formatting is quite strange. I would recommend doing it like this. 
```
<b-button class="exit-button" variant="danger" v-on:click="exitToHome">
	Exit Lobby
</b-button>
```

Goes for everything in this pull request.

File: frontend/src/modules/lobby/components/InvitationLink.vue [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+export default {
+  name: "InvitationLink",
+  data: function() {
+    return {
+      inviteLink: window.location.href
+    };
+  },
+  props: {
+    lobbyID: String,
+    stage: String
```
Comment (enochtsang) [inline frontend/src/modules/lobby/components/InvitationLink.vue:32]: I don't think is being used, can you take it out?

**Pull request merged**